src/Sql/Sql/ChangeLog.md

```diff
@@ -21,6 +21,9 @@
 
 Added `isManagedIdentityInUse` parameter for `Get-AzSqlServerMSSupportAudit`
 
+## Version 4.1.1
+* Added `PreferredEnclaveType` parameter to `NewAzureSqlDatabase`, `GetAzureSqlDatabase` and `SetAzureSqlDatabase` cmdlet
+
```

Comment:
Changes for the upcoming release should go under the section titled "Upcoming Release"
```suggestion
* Added `isManagedIdentityInUse` parameter for `Get-AzSqlServerMSSupportAudit`
* Added `PreferredEnclaveType` parameter to `NewAzureSqlDatabase`, `GetAzureSqlDatabase` and `SetAzureSqlDatabase` cmdlet

```